docs/_sass/_docs-display-component.scss

```diff
@@ -89,10 +89,6 @@
         width: 640px;
         height: 360px;
       }
-
-      .fd-calendar__navigation--main {
```

Comment:
You are removing the class but the .md still uses it

Reply:
This is documentation scss, the class is still there in the `calendar.scss`